lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
Minor: Since this is arguably presentational logic, did you consider making this a view helper method? I think I'm happy with the approach you've taken, because you've talked about ideally wanting to store the source data (as opposed to the partial template name) in the YAML.